packages/eslint-config-migme/preact.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
+  extends: [
+    './base',
+    './rules/strict',
+    './rules/preact',
```

Comment:
don't think it's able to put `react` and `preact` together in `extends`. but i got your point, it might be good to just reuse most of the code in `react.js`.  have an idea to import `react.js` in `preact.js` and modify part of the settings. lemme try it.